src/corefx/System.Globalization.Native/pal_collation.c

```diff
@@ -206,15 +178,11 @@ static UCharList* GetCustomRules(int32_t options, UColAttributeValue strength, i
             // Hiragana is the range 3041 to 3096 & 309D & 309E
             if (hiraganaChar <= 0x3096 || hiraganaChar >= 0x309D) // characters between 3096 and 309D are not mapped to katakana
```

Comment:
>0x309D [](start = 58, length = 6)

This should be 309F now?

Reply:
[https://unicode.org/charts/PDF/U3040.pdf](https://unicode.org/charts/PDF/U3040.pdf)

---
In reply to: [277039452](https://github.com/dotnet/coreclr/pull/24100#discussion_r277039452) [](ancestors = 277039452)

Reply:
Do you prefer me to update it here or in a separate PR?

Reply:
Maybe @eerhardt can comment why we didn't include 309F in the first place as this introduced awhile ago in Unicode 3.2. but you shouldn't block on this.